cli/cmd/sync.go

```diff
@@ -116,20 +123,37 @@ func syncConnection(ctx context.Context, pm *plugins.PluginManager, specReader *
 	format := " Syncing (%d resources) %s"
 	s.Suffix = fmt.Sprintf(format, 0, time.Duration(0))
 	s.Start()
-	failedWrites := 0
+	var failedWrites uint64 = 0
```

Comment:
```suggestion
	failedWrites := uint64(0)
```

Absolutely a nit, I just think this style fits better with the `totalResources := 0` declaration below it.